exporter/exporterhelper/internal/obs_report_sender.go

```diff
@@ -34,6 +39,9 @@ const (
 	ItemsSent = "items.sent"
 	// ItemsFailed used to track number of items that failed to be sent by exporters.
 	ItemsFailed = "items.failed"
+
+	// FailurePermanentKey indicates whether the error is permanent (non-retryable).
+	FailurePermanentKey = "failure.permanent"
```

Comment:
Should we use `error.permanent` for symmetry with the API, and since it's closer to `error.type`?

Perhaps we should also open a SemConv issue.

Reply:
Changed.

Created https://github.com/open-telemetry/semantic-conventions/issues/3227